io.github.gamingdoom.Datcord.yml

```diff
@@ -0,0 +1,85 @@
+app-id: io.github.gamingdoom.Datcord
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.mozilla.firefox.BaseApp
+base-version: '23.08'
+command: /app/lib/datcord/launch-datcord
+rename-icon: io.github.gamingdoom.Datcord
+add-build-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: '23.08'
+    autodownload: true
+    autodelete: false
+
+finish-args:
+    - --allow=devel                                           
+    - --share=ipc                                             
+    - --share=network                                         
+    - --env=GTK_PATH=/app/lib/gtkmodules                      
+    - --socket=pulseaudio                                     
+    - --socket=wayland                                        
+    - --socket=fallback-x11                                            
+    - --require-version=0.11.1                                
+    - --persist=.datcord                                      
+    - --filesystem=xdg-download:rw                            
+    - --filesystem=/run/.heim_org.h5l.kcm-socket              
+    - --filesystem=/tmp/open-in-default-browser                                       
+    - --device=all                                            
+    - --talk-name=org.freedesktop.FileManager1                
+    - --talk-name=org.a11y.Bus                                
+    - --talk-name=org.gnome.SessionManager                    
+    - --talk-name=org.freedesktop.ScreenSaver                 
+    - --talk-name=org.freedesktop.Notifications               
+    - --command=/app/lib/datcord/launch-datcord
+    - --filesystem=home:rw
```

Comment:
why do you need home access for a discord client?

Reply:
Home access is to create a .datcord directory to save firefox profiles containing the password for the discord account.

Reply:
You have `persist` above.

Better would be that your app follow the XDG spec.

Reply:
Upon further inspection, it works without home access so I have removed it.